pom.xml

```diff
@@ -1,211 +1,79 @@
 <project xmlns="http://maven.apache.org/POM/4.0.0"
-	xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
-	xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
-	<modelVersion>4.0.0</modelVersion>
-	<groupId>com.fleetpin</groupId>
-	<artifactId>graphql-dynamodb-manager</artifactId>
-	<version>0.1.3</version>
-	<name>GraphQL Builder</name>
-	<description>library to make working with dynamodb and graphql easy</description>
-	<url>https://github.com/fleetpin/graphql-dynamodb-manager</url>
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+    <groupId>com.fleetpin</groupId>
+    <artifactId>graphql-dynamodb-manager</artifactId>
```

Comment:
think we should rename the project to something like graphql-database-manager dynamo module name will be cleaner

Reply:
So would we rename this repo as well?